add-ons/hyper-sorter/index.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var _ = require('object-iterators');
+
+function Hypersorter(grid, objects) {
+    this.grid = grid;
+    objects = objects || {};
+
+    mixInTo('Hypergrid', grid, require('./mix-ins/grid'));
+    mixInTo('Behavior', grid.behavior, require('./mix-ins/behavior'));
+    mixInTo('Column', grid.behavior, require('./mix-ins/column'));
```

Comment:
Sorry I wasn't in the middle of changing this. I should held off on the PR a little longer